### DISCUSSION TITLE
docs: Authentication for forgejo/gitea 1.20

### DISCUSSION THREAD

**Discussion by morremeyer** [renovatebot/renovate#23625] [Request Help] [answered]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us what version of Renovate you run.

35.159.7, 36.25.5

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

The authentication for forgejo and gitea changed in 1.20, see the forgejo changelog: https://codeberg.org/forgejo/forgejo/src/branch/forgejo/RELEASE-NOTES.md#1-20-1-0

> [Scoped access tokens](https://codeberg.org/forgejo/forgejo/commit/18de83b2a3fc120922096b7348d6375094ae1532) or (Personal Access Tokens), were refactored and although existing tokens are still valid, they may have a different scope than before. To ensure that no tokens have a larger scope than expected they must be removed and recreated.

Existing tokens that were created with the `repo` permission will not have the `read:issue` permission anymore (and most likely neither the `write:issue` permission, but I revoked my PAT before I could test that).

The new settings look like this on forgejo

![image](https://github.com/renovatebot/renovate/assets/7683567/66d1039c-3d39-4a60-8816-b074397180a7)

We need to update the documentation to reflect this. I'll open a PR for this later this week.

**Comment by morremeyer** [comment, marked as answer]
This is resolved by https://github.com/renovatebot/renovate/pull/23626.

**Reply by HonkingGoose** [reply]
Or by:

- #23615
